OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -53,6 +53,7 @@ public void WorldLoaded(World w, WorldRenderer wr)
 			register("all", "toggles all cheats and gives you some cash for your trouble.");
 			register("crash", "crashes the game.");
 			register("levelup", "adds a specified number of levels to the selected actors.");
+			register("playerxp", "adds a specified number player experience to owners of selected actors.");
```

Comment:
The current wording doesn't make sense. Please change this to "adds a specified amount of player experience to the owner(s) of selected actors."